### PR TITLE
Fix cnrm instance commands

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @jlewi

### CODE REVIEW

File: management/README.md [hunk 1]
```diff
@@ -111,20 +111,20 @@ For each project you want to setup follow the instructions below.
 1. Create a copy of the per namespace/project resources
```
Comment (jlewi) [inline management/README.md:111]: Per: https://github.com/kubeflow/gcp-blueprints/issues/13 
I think we just want to install CNRM in non-namespace mode using workload identity. Would you mind changing this to something like  "follow the instructs https://cloud.google.com/config-connector/docs/how-to/install-upgrade-uninstall to install CNRM using Cloud Config Connector using workload identity in non namespace mode"

Comment (Bobgy) [reply management/README.md:111]: Got it, SGTM

Comment (jlewi) [reply management/README.md:111]: Are you planning on making any changes?

Comment (Bobgy) [reply management/README.md:111]: @jlewi Sorry, I've got some other urgent issues, won't have time to follow up on this one.

**Pull request merged**